examples/apps/heart_disease_eval/workload/heart_disease_evaluation_logic.cpp

```diff
@@ -91,27 +100,31 @@ double HeartDiseaseEvalLogic::score_cp(int cp_type) {
 }
 
 double HeartDiseaseEvalLogic::score_trestbps(double data) {
+    // Resting blood pressure (mm Hg)
     return model_A(218, 108, data);
 }
 
 double HeartDiseaseEvalLogic::score_chol(double data) {
+    // Serum cholesterol (mg/dl)
     return model_A(309, 126, data);
 }
 
 double HeartDiseaseEvalLogic::score_fbs(double data) {
+    // Fasting blood sugar (mg/dl)
     return model_A(248, 98, data);
 }
 
 double HeartDiseaseEvalLogic::score_restecg(int type) {
+    // Resting electrocardiographic results
     double score = 0.0;
     switch (type) {
-        case 0:
+        case 0: // normal
             score = 100.0;
             break;
-        case 1:
+        case 1: // ST-T wave abnormality
             score = 9.0;
             break;
-        case 2:
+        case 2: // hypertrophy
```

Comment:
We need a doctor to review this PR. I don't understand these terms much  :-)

Reply:
:-).
Julian Googled many of the variable names so he could name each variable in the GUI form.